packages/playwright-test/src/loader.ts

```diff
@@ -678,3 +679,19 @@ export function folderIsModule(folder: string): boolean {
   // Rely on `require` internal caching logic.
   return require(packageJsonPath).type === 'module';
 }
+
+function fixFilepathCapitalization(file: string): string {
+  /**
```

Comment:
should we a 
```ts
if (process.platform !== 'win32')
   return;
```
there?

Reply:
I'd call the method `fixWin32FilepathCapitalization` and call it on win32 platform only